typescript/cli/src/config/warp.ts

```diff
@@ -174,3 +195,23 @@ export function readWarpRouteConfig(filePath: string): WarpCoreConfig {
   if (!config) throw new Error(`No warp route config found at ${filePath}`);
   return WarpCoreConfigSchema.parse(config);
 }
+
+async function createDefaultWarpIsmConfig(
+  context: CommandContext,
+  remote: ChainName,
+  origins: ChainName[],
+) {
+  return {
+    type: IsmType.AGGREGATION,
+    modules: [
+      await createTrustedRelayerConfig(context),
+      await createRoutingConfig(
+        context,
+        IsmType.FALLBACK_ROUTING,
+        remote,
+        origins,
```

Comment:
I think we want origins to be empty here

Reply:
this is empty on cli-2.0– i think rebase was out of date when you saw this